WTWSMS/common/landed_titles/mercenaries.txt

```diff
@@ -2565,4 +2564,80 @@ d_ghanan_band = {
 	strength_growth_per_century = 1.0
```

Comment:
I didn't know that I was quoting Margeret Thatcher: "consensus is the absence of leadership" has long been one of my life's mottoes. You talked about "concertation", which seems to be a synonym of "conensus".